gendiff/formats/plain.py

```diff
@@ -0,0 +1,51 @@
+from typing import Any, Dict
+
+from gendiff.diff import get_diff_view
+
+
+def convert_value_plain_format(val: Any) -> Any:
+    exceptions_to_convert = {False: "false", True: "true", None: "null"}
+    if val in exceptions_to_convert.keys():
```

Comment:
`val in exceptions_to_convert` will do the same

Reply:
special thanks for the comment))
I've thought more and done really better)